src/core_ocean/shared/Makefile

```diff
@@ -66,13 +66,14 @@ OBJS = mpas_ocn_init_routines.o \
 	   mpas_ocn_sea_ice.o \
 	   mpas_ocn_framework_forcing.o \
 	   mpas_ocn_time_varying_forcing.o \
-	   mpas_ocn_wetting_drying.o
+	   mpas_ocn_wetting_drying.o \
+           mpas_ocn_tidal_potential_forcing.o
```

Comment:
Seems like there is a strange issue here with indenting...

Reply:
Thanks for catching that. I'll fix it.